### ISSUE TITLE
App deleted contents of 'Podcasts' folder upon first download.

### ISSUE THREAD

**Issue by zayuim** [bottiger/SoundWaves#153]
SoundWaves appears to delete any data in the Podcasts directory upon first downloading with SoundWaves. I used to folder to store other podcast's, is it possible to change this behavior?

**Comment by bottiger** [comment]
@zayuim This is definitely *not* intended behavior. I will investigate the issue ASAP

**Comment by bottiger** [comment]
@zayuim Are you reproduce this again? I am not able to reproduce it myself, and I do not know how this could happen

**Comment by zayuim** [comment]
@bottiger No, for some reason it only happened once. All I can tell you is that it was the first download I did with the app, And the download did pause, and I think it may of failed (connection trouble).
